elastalert/kibana.py

```diff
@@ -166,6 +166,8 @@
                   u'style': u'dark',
                   u'title': u'ElastAlert Alert Dashboard'}
 
+kibana4_time_temp = "(refreshInterval:(display:Off,section:0,value:0),time:(from:'%s',mode:absolute,to:'%s'))"
```

Comment:
I think it should default to no autorefresh. I personally find it obnoxious sometimes, and with a set time window, these dashboards are likely to not change after an alert fires unless data is being backfilled. Having the option would be nice though.